alerting/src/test/kotlin/org/opensearch/alerting/AlertingRestTestCase.kt

```diff
@@ -69,6 +69,9 @@ abstract class AlertingRestTestCase : ODFERestTestCase() {
     protected val numberOfNodes = System.getProperty("cluster.number_of_nodes", "1")!!.toInt()
     protected val isMultiNode = numberOfNodes > 1
 
+    protected val statsResponseOpendistroSweeperEnabledSetting = "opendistro.scheduled_jobs.enabled"
+    protected val statsResponseOpenSearchSweeperEnabledSetting = "plugins.scheduled_jobs.enabled"
```

Comment:
nit: should we rather call it ``Field`` instead of ``Setting``, given this is part of api response structure.

Reply:
I guess `statsResponseOpenSearchSweeperEnabledSettingField` is the **most** accurate since it is just the setting that we're outputting here but it's quite verbose. Do you have a preference between that and just `Field`?

Reply:
Either way it looks good. Only keep it less verbose I might just keep ``Field``